src/components/forms/lib/handleDemoSubmit.ts

```diff
@@ -49,7 +52,7 @@ const getUrl = async ({ isCompany, size, email }: ParsedFormValues) => {
   if (alwaysBook) return meetingRequestUrlWithEmail;
 
   if (!isCompany) {
-    return isFund(size) ? meetingRequestUrlWithEmail : investorUrl;
+    return investorUrl;
```

Comment:
This seems to be a constant. To be consistent with our naming convention should name it
```suggestion
    return INVESTOR_URL;
```